Philips.CodeAnalysis.Test/MsTest/NoEmptyTestMethodsDiagnosticAnalyzerTest.cs

```diff
@@ -24,9 +25,9 @@ protected override DiagnosticAnalyzer GetDiagnosticAnalyzer()
 			return new NoEmptyTestMethodsDiagnosticAnalyzer();
 		}
 
-		protected override MetadataReference[] GetMetadataReferences()
+		protected override ImmutableArray<MetadataReference> GetMetadataReferences()
```

Comment:
Ditto. I think this override can be deleted.

Reply:
Removed